package.json

```diff
@@ -201,7 +201,7 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "^1.0.0",
+    "@ciscospark/eslint-config": "^2.0.0",
```

Comment:
The reason I had to go to 2.0 is due to the fact that npm would not let me publish unless I bumped up the version.